ftw/collectionblock/profiles/default_plone5/registry.xml

```diff
@@ -2,8 +2,8 @@
 
     <record name="plone.types_not_searched"
             interface="Products.CMFPlone.interfaces.controlpanel.ISearchSchema"
```

Comment:
I think you shouldn't need purge=false twice..... Looking at ftw.simplelayout this way seems adequate:

```
  <records interface="Products.CMFPlone.interfaces.controlpanel.ISearchSchema"
           prefix="plone">
    <value key="types_not_searched" purge="false">
           ...
```

Maybe try this approach in the uninstall profile?